src/components/Tokens/TokenTable/TokenTable.tsx

```diff
@@ -92,15 +128,39 @@ export default function TokenTable() {
     } else {
       return (
         <>
-          <GridContainer>
+          <GridContainer fixedHeight={true}>
```

Comment:
can drop `={true}` and just name the prop for the same effect

Reply:
sgtm done